pallets/unique/src/lib.rs

```diff
@@ -983,18 +983,33 @@ decl_module! {
 			})
 		}
 
-		/// Repairs a broken item
+		/// Repairs a collection's properties if the data was somehow corrupted.
```

Comment:
Мы можем использовать в будущем и для других починок

```suggestion
		/// Repairs a collection if the data was somehow corrupted.
```

Reply:
Да, но я рассчитывал, что в будущем и документацию обновим) Но ладно